dolosse/run_control/cli/kafka_pyinq.py

```diff
@@ -0,0 +1,263 @@
+"""
+file: midas_kafka_control_producer
+brief: command line interface using PyInquirer for producing control commands to kafka
+author: Caswell Pieters
+date: 24 March 2020
+"""
+
+from __future__ import print_function, unicode_literals

```

Comment:
I'm still on the fence about futures. This is *unreleased* functionality that will be included in a _future_ python version.

Reply:
Removed.